service/key/key.go

```diff
@@ -24,6 +24,24 @@ func AzureCloudType(customObject providerv1alpha1.AzureConfig) string {
 	return defaultAzureCloudType
 }
 
+func AdminUsername(customObject providerv1alpha1.AzureConfig) string {
+	users := customObject.Spec.Cluster.Kubernetes.SSH.UserList
+	// We don't want panics when someone is doing something nasty.
+	if len(users) == 0 {
+		return ""
+	}
+	return users[0].Name
```

Comment:
Why is admin the first?

Reply:
Because there is no particular preference who should be that one. And there is an requirement that someone has to be. First one is the most natural choice for me in this case.